src/Microsoft.DotNet.Build.Tasks/PackageFiles/test-runtime/project.json

```diff
@@ -1,33 +0,0 @@
-{
-  "dependencies": {
```

Comment:
We should just delete this. I know we've mentioned that this file is unused in passing a few times. It's just confusing people, so let's delete it.